### PR TITLE
PMM-13020 Drop identifier prefixes

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-13020

Link to the Feature Build: SUBMODULES-3625

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [x] API Docs updated

### CODE REVIEW

**Review by BupycHuk — COMMENTED** [review]
looks good so far,
what do you think about compatibilty mode?
it's for pmm-agents which still has `/agent_id/` prefix and for users who has automations to communicate to PMM API

**Review by JiriCtvrtka — APPROVED** [review]

File: agent/config/config.go [hunk 1]
```diff
@@ -196,6 +197,10 @@ func get(args []string, cfg *Config, l *logrus.Entry) (string, error) { //nolint
 		}
 
 		// set default values
+		if strings.HasPrefix(cfg.ID, agentPrefix) {
+			l.Warnf("The agent ID '%s' contains a legacy prefix '%s'. It will be used without it.", cfg.ID, agentPrefix)
+			cfg.ID, _ = strings.CutPrefix(cfg.ID, agentPrefix)
+		}
```
Comment (BupycHuk) [inline agent/config/config.go:203]: 👍🏼

**Review by BupycHuk — APPROVED** [review]
Great job, can we create a task for compatible mode?

**Comment by ademidoff** [comment]
> Great job, can we create a task for compatible mode?

Sure, will do right away!

**Pull request merged**